pkg/command/context.go

```diff
@@ -1535,7 +1535,9 @@ func setTanzuCtxActiveResource(_ *cobra.Command, args []string) error {
 	if spaceStr != "" && clustergroupStr != "" {
 		return errors.Errorf("either space or clustergroup can be set as active resource. Please provide either --space or --clustergroup option")
 	}
-	// TODO(prkalle): Add condition to check projectID is also set if project name set
+	if (projectStr != "" && projectIDStr == "") || (projectStr == "" && projectIDStr != "") {
+		return errors.Errorf("both project name and project ID are required to set a project as active. Please provide them using --project and --project-id options")
```

Comment:
Older plugins that only specify the project name will not work with this version of the CLI.
So this breaks backwards-compatibility.
I assume we can accept that because the plugins are still beta?

Reply:
Yes, that's correct. This is a breaking change and plugins should update to use new plugin-runtime and new CLI version to work. Yes,It should be acceptable as the plugins are still in beta.